From 23997830a5e0078c8bae0f8baefe6129f4efdab6 Mon Sep 17 00:00:00 2001
From: Varun Saxena <varunsaxena@apache.org>
Date: Thu, 27 Oct 2016 13:41:32 +0530
Subject: [PATCH 2153/2848] YARN-5752.
 TestLocalResourcesTrackerImpl#testLocalResourceCache
 times out (Eric Badger via Varun Saxena)

(cherry picked from commit 5c39a419a6429a6ec9d510a57c19a04808d3b9cc)

Change-Id: Ib5c6cf6b19c1983d513c64a5c68449336203a118
---
 .../localizer/TestLocalResourcesTrackerImpl.java   |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/TestLocalResourcesTrackerImpl.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/TestLocalResourcesTrackerImpl.java
index 8cf3b2e..efc150e 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/TestLocalResourcesTrackerImpl.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/localizer/TestLocalResourcesTrackerImpl.java
@@ -236,7 +236,7 @@ public void testConsistency() {
     }
   }
 
-  @Test(timeout = 1000)
+  @Test(timeout = 10000)
   @SuppressWarnings("unchecked")
   public void testLocalResourceCache() {
     String user = "testuser";
@@ -381,7 +381,7 @@ public void testLocalResourceCache() {
     }
   }
 
-  @Test(timeout = 100000)
+  @Test(timeout = 10000)
   @SuppressWarnings("unchecked")
   public void testHierarchicalLocalCacheDirectories() {
     String user = "testuser";
-- 
1.7.9.5

